### ISSUE TITLE
Vigilante8 wrong menu transitions

### ISSUE THREAD

**Issue by purplemarshmallow** [gonetz/GLideN64#761]
from #724 
>Vig 8 seems to switch resolutions when it performs the fullscreen flip
(Images taken from Angrylion's.)

![f8fab224-66c4-11e5-93a9-b3b9394525ae](https://cloud.githubusercontent.com/assets/7283660/10540868/537db59e-740c-11e5-83e5-c804c373cc1f.jpg)
![f902737e-66c4-11e5-821b-c5ef7b3c9963](https://cloud.githubusercontent.com/assets/7283660/10540869/53811f9a-740c-11e5-8f75-cd762b9fb9d4.jpg)

**Comment by AmbientMalice** [comment]
z64gl seems to be able to do the screen flipping despite not having FB to RDRAM support, AFAIK.

**Comment by olivieryuyu** [comment]
correct, VI_H_REG_WITH switches to 0x140. The framebuffer goes from 640x480 to 320x240.

**Comment by AmbientMalice** [comment]
Issue partially fixed in latest versions with Aux FB to RAM. But graphics are prone to corruption when switching between menus.

**Comment by GamingTVYT** [comment]
Would it be hard work fixing this graphic problem? V8 is a great game of the N64!
